theme/static/js/group.js

```diff
@@ -89,6 +89,7 @@ var colDefs = [
 ];
 
 function group_invite_ajax_submit() {
+    alert("dev2");
```

Comment:
This should be removed.

Reply:
Quite right thank you. Apparent that I needed to slow down so I removed this and also added some additional tests and a check for explanation string length